csrc/ir/internal_nodes.h

```diff
@@ -3103,6 +3103,150 @@ class GroupedMmaOp : public Expr {
   IterDomain* getGroupDimOfOutput() const;
 };
 
+//! NOTE -- [ Scaled Matrix Multiplication semantics ]
+//!
+//! This operation performs a grouped matrix multiplication.
```

Comment:
```suggestion
//! This operation performs a scaled matrix multiplication.
```

Reply:
oops, let me do another search on the diff.